lib/session.js

```diff
@@ -46,6 +114,16 @@ Session.prototype._getSshConnInfo = function() {
 };
 
 Session.prototype._withSshClient = function(callback) {
+  var self = this;
+
+  if(this._ssh_config === null) {
```

Comment:
Hey, don't write code like this. Hope you are kidding here.
This is a loop bro.